product_variant_supplierinfo/models/procurement.py

```diff
@@ -15,13 +15,14 @@ def _get_product_supplier(self, procurement):
         """returns the main supplier of the procurement's product
            given as argument"""
         product = procurement.product_id
-        company_supplier = self.env['product.supplierinfo'].search([
-            '|',
-            '&',
-            ('product_tmpl_id', '=', product.product_tmpl_id.id),
-            ('product_id', '=', False),
-            ('product_id', '=', product.id),
-            ('company_id', '=', procurement.company_id.id),
+        company_supplier = self.env['product.supplierinfo'].search(
```

Comment:
Hi @pedrobaeza, Il change this because I got a PEP 8 error (closing bracket does not match indentation)